packages/schematics/angular/application/files/standalone-files/src/app/app.config.ts.template

```diff
@@ -4,5 +4,8 @@ import { provideRouter } from '@angular/router';
 import { routes } from './app.routes';<% } %>
 
 export const appConfig: ApplicationConfig = {
-  providers: [provideZoneChangeDetection({ eventCoalescing: true })<% if (routing) { %>, provideRouter(routes)<% } %>]
+  providers: [
+  <% if(experimentalZoneless) { %>provideExperimentalZonelessChangeDetection(),<% } %>
+  provideZoneChangeDetection({ eventCoalescing: true })<% if (routing) { %>, provideRouter(routes)<% } %>
```

Comment:
`provideZoneChangeDetection` is still added when using zoneless. This should not be the case.